### ISSUE TITLE
"Open query in neo4j" button

### ISSUE THREAD

**Issue by vincerubinetti** [greenelab/hetmech-frontend#49]
Button to open the same source/target and metapaths query in the neo4j browser view.

**Comment by dhimmel** [comment]
Yeah, the backend should probably pass these queries back with the query-metapaths results.

**Comment by dhimmel** [comment]
The query-metapaths API endpoint now returns a `cypher_query` field. See https://search-api.het.io/v1/query-metapaths/?source=17599&target=31491 for example:

```json
            "cypher_query": "MATCH path = (n0:Gene)-[:PARTICIPATES_GpPW]-(n1)-[:PARTICIPATES_GpPW]-(n2)-[:INTERACTS_GiG]-(n3:Gene)\nUSING JOIN ON n1\nWHERE n0.identifier = 675 // BRCA2\nAND n3.identifier = 672 // BRCA1\nAND n0 <> n2 AND n0 <> n3 AND n2 <> n3\nWITH\n[\nsize((n0)-[:PARTICIPATES_GpPW]-()),\nsize(()-[:PARTICIPATES_GpPW]-(n1)),\nsize((n1)-[:PARTICIPATES_GpPW]-()),\nsize(()-[:PARTICIPATES_GpPW]-(n2)),\nsize((n2)-[:INTERACTS_GiG]-()),\nsize(()-[:INTERACTS_GiG]-(n3))\n] AS degrees, path\nWITH path, reduce(pdp = 1.0, d in degrees| pdp * d ^ -0.5) AS PDP\nWITH collect({paths: path, PDPs: PDP}) AS data_maps, sum(PDP) AS DWPC\nUNWIND data_maps AS data_map\nWITH data_map.paths AS path, data_map.PDPs AS PDP, DWPC\nRETURN\n  path AS neo4j_path,\n  substring(reduce(s = '', node IN nodes(path)| s + '–' + node.name), 1) AS path,\n  PDP,\n  100 * (PDP / DWPC) AS percent_of_DWPC\nORDER BY percent_of_DWPC DESC\nLIMIT 10"
```

If my understanding of https://github.com/neo4j/neo4j-browser/pull/153 is correct, it won't be possible to pass the cypher to a neo4j browser using a URL parameter. Therefore, perhaps a copy to clipboard button would be the next best thing?

**Comment by vincerubinetti** [comment]
closed by #71

**Comment by dhimmel** [comment]
Should we disable sorting on the "neo4j query" column? When I click the column header to open the neo4j browser, it ends up sorting (out of view because have been relocated to new tab).

**Comment by vincerubinetti** [comment]
> Should we disable sorting on the "neo4j query" column? When I click the column header to open the neo4j browser, it ends up sorting (out of view because have been relocated to new tab).

Added fix for this to PR #77 . I still keep the column sortable, but when user clicks directly on link, the click doesn't also propagate to the sort button "below" it.
